src/client/datascience/jupyter/kernelVariables.ts

```diff
@@ -35,7 +35,15 @@ const DocStringRegex = /.*?\[.*?;31mDocstring:.*?\[0m\s+(.*)/;
 const CountRegex = /.*?\[.*?;31mLength:.*?\[0m\s+(.*)/;
 const ShapeRegex = /^\s+\[(\d+) rows x (\d+) columns\]/m;
 
-const DataViewableTypes: Set<string> = new Set<string>(['DataFrame', 'list', 'dict', 'ndarray', 'Series', 'Tensor']);
+const DataViewableTypes: Set<string> = new Set<string>([
+    'DataFrame',
+    'list',
+    'dict',
+    'ndarray',
+    'Series',
+    'Tensor',
+    'EagerTensor'
```

Comment:
Not sure what the best way to feature gate this is. The problem is we can only query experiment status from TS code. I have 2 ideas:
1. Add 'Tensor' and 'EagerTensor' to the set of data viewable types only if the user is in the experiment. This will stop the user from viewing tensors in the data viewer
2. Create a completely separate Python script codepath that is invoked from TS based on whether we're in the experiment (I don't like this one much, adds lots of complexity)

Thoughts?

Reply:
I don't think we should bother putting this in an experiment. I think it should just work.

Reply:
Otherwise I just pick option 1. Have the kernelVariables class not allow viewing them.

Reply:
jmew says we can skip the experiment here 🎉